gateway/1.7/modules/ROOT/pages/flex-troubleshoot-linux-permissions.adoc

```diff
@@ -0,0 +1,53 @@
+= Troubleshooting Linux Permissions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+In Linux, to limit access to information about a Flex Gateway, the following `flexctl` commands require that user belong to the `flex` user group or to be the root user to execute them:
+
+* `flexctl dump`
+* `flexctl version`
+* `flexctl check connections`
+* `flexctl probe`
+* `flexctl registration inspect` (when ran without the `--file` flag)
+
+There are two options to execute these commands:
+
+* <<add-user-to-flex-group>>
+* <<run-as-root-user>>
+
+[[add-user-to-flex-group]]
+== Add the User to the Flex User Group
+
+Execute the following command to add the current user to the `flex` user group:
+
+[source,ssh]
+----
+sudo usermod -a -G flex $USER
+----
+
+After running the command, the user must log in to the VM again to begin using the `flexctl` commands as normal.
+
+[[run-as-root-user]]
+== Run as the Root User
+
+Execute the `sudo` command with your `flexctl` command:
+
+[source,ssh]
+----
+sudo flexctl <command>
+----
+
+Executing the `sudo` command with your `flexctl` command is the quickest way to access Flex Gateway information. However, you must execute the `sudo` command everytime you run a `flexctl` command.
+
+When running the `flexctl dump` command, the root user is the owner of the created dump files. If the current user is not the root user, the user cannot view the dumped files. To view the files, execute the following command to change the owner of the files:
```

Comment:
The intro clause doesn't quite work with the sentence, because it isn't followed by who is doing the running. How about something like:
When the `flexctl dump` command is executed, ...

Reply:
Or you could say "When you run the ... command, ..." if you want to keep it active.